### PR TITLE
Building a custom image for system with Oracle Database support

### PR DESCRIPTION
**Pull request by hallelujah**
Needed by https://github.com/3scale/openshift-templates/pull/300

### CODE REVIEW

File: amp/system-oracle/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# System image build with Oracle Database support
+
+
+## Prerequisites
+
+In order to build the image with Oracle Database support,
+you will need to download and add the binaries inside the directory `oracle-client-files`.
```
Comment (andrewdavidmackenzie) [inline amp/system-oracle/README.md]: I'd clarify that this is a directory on your local machine where you will have to run the commands further down.

File: amp/system-oracle/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Download the bellow instant clients files from the version **12.2** of oracle.
```
Comment (andrewdavidmackenzie) [inline amp/system-oracle/README.md]: below

"Download the three "Instant Client Package" files listed below to the "oracle-client-file" on your local machine"

File: amp/system-oracle/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    * instantclient-basiclite-linux.x64-12.2.0.1.0.zip or instantclient-basic-linux.x64-12.2.0.1.0.zip
+    * instantclient-sdk-linux.x64-12.2.0.1.0.zip
+    * instantclient-odbc-linux.x64-12.2.0.1.0-2.zip
+
+## Instructions
+
+* Get the productized amp.yml template
+* Install the binary files as instructed above
+* Run
```
Comment (andrewdavidmackenzie) [inline amp/system-oracle/README.md]: "From the terminal, while in the "oracle-client-files" directory, run the following commands:

File: amp/system-oracle/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+```shell
+$ oc new-app -f build.yml -p AMP_RELEASE=2.2.0-CR1
+$ oc new-app -f amp.yml -p DATABASE_URL="oracle-enhanced://user:password@my-oracle-database.com:1521/threescalepdb" -p WILDCARD_DOMAIN=mydomain.com
```
Comment (maneta) [inline amp/system-oracle/README.md]: Actually, we don't have this `DATABASE_URL` in the template, we must provide more instructions/documentation on how to set this configuration.

File: amp/system-oracle/README.md (hunk 3 above)
Comment (hallelujah) [reply amp/system-oracle/README.md]: Not really in the local oracle-client-files but OK I get it

**Pull request merged**